chpl/chpl-service/src/main/java/gov/healthit/chpl/domain/CacheStatusName.java

```diff
@@ -0,0 +1,5 @@
+package gov.healthit.chpl.domain;
+
+public enum CacheStatusName {
+    OK, PENDING;
```

Comment:
I saw something in a comment recently about enums and hibernate (I think). We needed to do something to make enums work correctly somewhere? Does that ring a bell?

Reply:
There is something you have to do to make an enum work with hibernate and we have done it with a few enums in the past but this class is in the domain package (used to send data back and forth to the front end) and won't ever touch the database.